Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
 RUN pip install mozilla-django-oidc
 RUN pip install spam_patterns@git+https://github.com/CUAHSI/spam_patterns.git@0.0.4
 RUN pip install freezegun
+RUN pip install --upgrade hsmodels
+RUN pip install -U email-validator
```

Comment:
why `email-validator` ?

Reply:
I ran into a version mismatch with pydantic and an older version of email-validator. Updating email-validator resolved the issue.